lib/mailer/sfMailerSwiftMessage.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the symfony package.
+ * (c) Fabien Potencier <fabien.potencier@symfony-project.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+class_exists('Swift');
+
+if (version_compare(Swift::VERSION, '6.0.0') >= 0) {
```

Comment:
This approach is a bit like https://github.com/FriendsOfSymfony1/symfony1/pull/237/files but way more elegant. 

Question is, if we are able to support swiftmailer 6 including the doctrine spools, do we really need to support swiftmailer 5?

Reply:
Good question, I guess not.

But it depends, on the ability of applications using swiftmailer 5 to migrate to swiftmailer 6.

Reply:
This opens a path to add support for `symfony/mailer`.